plugins/python-build/bin/python-build

```diff
@@ -995,7 +995,13 @@ build_package_activepython() {
 
 anaconda_architecture() {
   case "$(uname -s)" in
-  "Darwin" ) echo "MacOSX-x86_64" ;;
+  "Darwin" )
+    case "$(uname -m)" in
+    "arm64" ) echo "MacOSX-arm64" ;;
```

Comment:
Is it safe to assume that macOS users on ARM always want the ARM version? If some dependencies do not support ARM, they may prefer x86_64 (executed via Rosetta).

Reply:
Hi @chrahunt, the approach I suggested is safe and here's why: macOS users on ARM can run programs in x86_64  mode on Rosetta 2 translator quite simple and transparent with the command `arch x86_64`. For example, `arch x86_64 pyenv install 3.9.1` build and install python for x86_64 architecture. In this way, the "arch" command runs commands under Rosetta 2. This approach is checked and tested on my MacBook Air M1 - everything works well! In fact, this is the only (correct) approach, that supported by Apple, and, may be, it would be incorrect to add any new parameters for pyenv, such as `pyenv install 3.9.1 x86_64`ю

Reply:
Thank you for the explanation! I agree, in that light this seems like the right approach.